Directory.Build.targets

```diff
@@ -21,4 +21,15 @@
     -->
     <NETCoreAppMaximumVersion>$(MajorVersion).$(MinorVersion)</NETCoreAppMaximumVersion>
   </PropertyGroup>
+
+  <!-- Packaging -->
+  <PropertyGroup>
+    <PackageDescription Condition="'$(PackageDescription)' == '' and '$(Description)' != ''">$(Description)</PackageDescription>
+    <PackageDescription Condition="'@(PackageDescriptionCommonType)' != '' and '$(PackageDescription)' != ''">$(PackageDescription)%0A%0ACommonly Used Types:@(PackageDescriptionCommonType->'%0A%(Identity)')</PackageDescription>
```

Comment:
This won't work because of evaluation order. I will see if there's a way outside of a target but I doubt so.